tests/openebs-pool-container-failure_test.go

```diff
@@ -80,10 +82,41 @@ var _ = BeforeSuite(func() {
 var _ = Describe("BDD of openebs pool container failure experiment", func() {
 
 	// BDD TEST CASE 1
+	resourceVersionSumBefore := 0
+	restartCountSumBefore := 0
 	Context("Check for the components of openebs", func() {
 
 		It("Should check for creation of runner pod", func() {
 
+			//Getting the Sum of Resource Version before Chaos
+			csp, err := client.CoreV1().Pods("openebs").List(metav1.ListOptions{LabelSelector: "app=cstor-pool"})
+			Expect(err).To(BeNil(), "fail to get csp pods")
+			for _, podSpec := range csp.Items {
+				i1, _ := strconv.Atoi(podSpec.ResourceVersion)
+				resourceVersionSumBefore = resourceVersionSumBefore + i1
+			}
+
+			fmt.Printf("Resource Version before chaos has been recorded\n")
+
+			//Getting the ContainerIDs of CSP pod Containers
+			k := 0
+			for _, podSpec := range csp.Items {
+				for i := 0; i < 3; i++ {
+					containeridbefore[k] = (podSpec.Status.ContainerStatuses[i].ContainerID)
+					k++
+				}
+			}
```

Comment:
for _, podSpec := range csp.Items {
				for i := 0; i < 3; i++ {
					container_id_before[k] = (podSpec.Status.ContainerStatuses[i].ContainerID)
                                        restartCountSumBefore = restartCountSumBefore + int(podSpec.Status.ContainerStatuses[i].RestartCount)
					k++
				}
			}